README.md

```diff
@@ -1,2 +1,36 @@
-# magento-payment-module
-Nelo Payment module for Magento
+# Nelo Payment Module for Magento
+Use the Nelo module for Magento and offer users to buy now and pay later.
+
+## How it works
+After install our module and enable it for your store with all required configuration, 
+the user will see Nelo promotional message as a payment option.  
+
+After user clics on "Continue to Nelo" button, he will be redirected to our web checkout
+and apply for a credit. Whether or not the credit is approved, when finished the user 
+will be redirected back to the corresponding page in your store.
+
+### Order states
+With our module when an order is placed the order is created with `pending_payment` state.
+If an error occurs during the process, the credit wasn't approved or the user cancels 
```

Comment:
If either an error occurs during the checkout flow, the credit isn't approved, or the user cancels the flow, then the order will be moved to the `canceled` state.